resdk/resources/data.py

```diff
@@ -48,171 +45,132 @@ def __init__(self, resolwe, **model_data):
         """Initialize attributes."""
         self.logger = logging.getLogger(__name__)
 
-        #: list of ``Collection``s that contain ``Data`` (lazy loaded)
-        self._collections = None
-        #: ``DescriptorSchema`` id of ``Data`` object
-        self._descriptor_schema = None
+        #: ``Collection``s that contains ``Data``
+        self._collection = None
         #: ``DescriptorSchema`` of ``Data`` object
-        self._hydrated_descriptor_schema = None
-        #: ``Sample`` containing ``Data`` object (lazy loaded)
+        self._descriptor_schema = None
+        #: The process used in this data object
+        self._process = None
+        #: ``Sample`` containing ``Data`` object
         self._sample = None
+
         #: ``ResolweQuery`` containing parent ``Data`` objects (lazy loaded)
         self._parents = None
         #: ``ResolweQuery`` containing child ``Data`` objects (lazy loaded)
         self._children = None
 
-        #: specification of inputs
-        self.process_input_schema = None
-        #: actual input values
-        self.input = None
-        #: specification of outputs
-        self.process_output_schema = None
-        #: actual output values
-        self.output = None
-        #: annotation data, with the form defined in descriptor_schema
-        self.descriptor = None
-        #: The ID of the process used in this data object
-        self.process = None
         #: checksum field calculated on inputs
         self.checksum = None
-        #: process status - Possible values: Uploading(UP), Resolving(RE),
-        #: Waiting(WT), Processing(PR), Done(OK), Error(ER), Dirty (DR)
-        self.status = None
-        #: process progress in percentage
-        self.process_progress = None
-        #: Process algorithm return code
-        self.process_rc = None
-        #: info log message (list of strings)
-        self.process_info = None
-        #: warning log message (list of strings)
-        self.process_warning = None
-        #: error log message (list of strings)
-        self.process_error = None
-        #: what kind of output does process produce
-        self.process_type = None
-        #: process name
-        self.process_name = None
-        #: data object's tags
-        self.tags = None
         #: indicate whether `descriptor` doesn't match `descriptor_schema` (is dirty)
         self.descriptor_dirty = None
-        #: process slug
-        self.process_slug = None
+        #: annotation data, with the form defined in descriptor_schema
```

Comment:
Is there any reason for reordering this in this commit?

Reply:
No, should be put into a separate one.